kapitan/cli.py

```diff
@@ -196,7 +197,17 @@ def main():
         parser.print_help()
         sys.exit(1)
 
+    if hasattr(args, 'verbose') and args.verbose:
+        logging.basicConfig(level=logging.DEBUG,
+                            format='%(asctime)s %(name)-12s %(levelname)-8s %(message)s')
+    elif hasattr(args, 'quiet') and args.quiet:
+        logging.basicConfig(level=logging.CRITICAL, format="%(message)s")
+    else:
+        logging.basicConfig(level=logging.INFO, format="%(message)s")
+
     if cmd == 'eval':
+        logging.basicConfig(level=logging.WARNING)
```

Comment:
Why warning for eval?

Reply:
It was set to warning previously as well (that's the default level for the python logging module).
All other operations were defaulting to INFO or DEBUG except eval. So I thought that was by intention and kept it the same. I could also just keep it at INFO when using eval if we prefer that(?).

Reply:
Yea we should use INFO by default, not sure why eval was at WARNING, maybe @ramaro might know more

Reply:
Not sure why it is WARNING 🤔  We definitely should use INFO